priv/shell-completion/fish/rebar3.fish

```diff
@@ -158,7 +157,6 @@ complete -f -c 'rebar3' -n '__fish_rebar3_needs_command paths' -s s -l separator
 complete -f -c 'rebar3' -n '__fish_rebar3_needs_command paths' -l src -d "Return the `src` path of the current profile`s applications."
 complete -f -c 'rebar3' -n '__fish_rebar3_needs_command paths' -l rel -d "Return the `rel` path of the current profile."
 
-complete -f -c 'rebar3' -n '__fish_rebar3_needs_command' -a pkgs -d "List available packages."
```

Comment:
`complete -f -c 'rebar3' -n '__fish_rebar3_needs_command' -a pkgs -d "List information for a hex package."`